index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins:400,700,900' rel='stylesheet'>
+  </head>
+  <body>
```

Comment:
- [ ] Refer to the following image. The webpage is supposed to fill the mobile size on chrome dev tools, but it corners as you decrease the size of the screen. Kindly fix that.

![image](https://user-images.githubusercontent.com/92215878/191767552-458508fe-76a8-4ffb-9b70-c59a73722420.png)